specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2019-12-01-preview/containerregistry.json

```diff
@@ -15,6 +15,178 @@
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ExportPipelineGet": {
+            "$ref": "./examples/ExportPipelineGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ExportPipelines"
+        ],
+        "description": "Creates an export pipeline for a container registry with the specified parameters.",
+        "operationId": "ExportPipelines_Create",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/RegistryNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ExportPipelineNameParameter"
+          },
+          {
+            "name": "exportPipelineCreateParameters",
+            "in": "body",
+            "description": "The parameters for creating an export pipeline.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ExportPipeline"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the operation has completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/ExportPipeline"
+            }
+          },
+          "201": {
+            "description": "The request was successful; the operation will complete asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/ExportPipeline"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ExportPipelineCreate": {
+            "$ref": "./examples/ExportPipelineCreate.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "delete": {
+        "tags": [
+          "ExportPipelines"
+        ],
+        "description": "Deletes an export pipeline from a container registry.",
+        "operationId": "ExportPipelines_Delete",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/RegistryNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ExportPipelineNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the request was well-formed and received properly."
+          },
+          "202": {
+            "description": "The request was successful; the operation will complete asynchronously."
+          },
+          "204": {
+            "description": "The export pipeline does not exist in the subscription."
```

Comment:
>subscription [](start = 70, length = 12)

Should probably say "resource group".

Reply:
@majastrz this phrasing is uniform with out other resource 204 messages in this swagger. Shall we update the existing ones as well?

Reply:
Up to you, but on 2nd thought I'm ok with current phrasing as well. It's less specific than what I proposed, but it's still accurate.

---
In reply to: [419804172](https://github.com/Azure/azure-rest-api-specs/pull/8662#discussion_r419804172) [](ancestors = 419804172)